tsconfig.eslint.json

```diff
@@ -1,4 +1,5 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["**/*.ts"]
+  "include": ["**/*.ts"],
+  "files": ["fake.ts"]
```

Comment:
do we need this?

Reply:
Yes, otherwise an error is thrown when running tests:
` A fatal parsing error occurred: Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
    The file does not match your project config: fake.ts.
    The file must be included in at least one of the projects provided.`

Reply:
👍
Weird I would've expected that the file should exist.

Reply:
@jsaguet could we create an eslint config in the test folder and use that on in our tests? It can be a simple config, something simalar to the [rxjs eslint plugin](https://github.com/cartant/eslint-plugin-rxjs/tree/master/tests)

Reply:
I don't understand why, but I didn't manage to make it work with such a file